actors/init/src/main/java/org/sunbird/init/dao/impl/SystemSettingDaoImpl.java

```
@@ -0,0 +1,66 @@
+
+/**
+ * This class implements the cassandra db operation DAO methods (insert,read) for system settings
+ *
+ * @author Loganathan
+ */
+public class SystemSettingDaoImpl implements SystemSettingDao {
+
+  private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+  private ObjectMapper mapper = new ObjectMapper();
+  private static final String KEYSPACE_NAME = JsonKey.SUNBIRD;
+  private static final String TABLE_NAME = JsonKey.SYSTEM_SETTINGS_DB;
+
+  /**
+   * This methods inserts the given settings record into cassandra table through CassandraOperation
+   * methods
+   *
+   * @param systemSetting instance of SystemSetting class conatins elements like id,field,value
+   * @return response instance of Response class returned by CassandraOperation insert method
+   */
+  @Override
+  public Response write(SystemSetting systemSetting) {
+    Map<String, Object> map = mapper.convertValue(systemSetting, Map.class);
+    Response response = cassandraOperation.insertRecord(KEYSPACE_NAME, TABLE_NAME, map);
+    response.put(JsonKey.ID, map.get(JsonKey.ID));
+    return response;
+  }
+
+  /**
+   * This methods fetch the settings record using given id from cassandra table through
+   * CassandraOperation methods
+   *
+   * @param id id of the settings record to be fetched
+   * @return instance of SystemSetting class with mapped field values(id,field,value) from cassandra
+   *     table
+   */
+  @Override
+  public SystemSetting readById(String id) {
+    Response response = cassandraOperation.getRecordById(KEYSPACE_NAME, TABLE_NAME, id);
+    List<Map<String, Object>> list = (List<Map<String, Object>>) response.get(JsonKey.RESPONSE);
+    if (CollectionUtils.isEmpty(list)) {
+      return null;
+    }
+    try {
+      String jsonString = mapper.writeValueAsString((Map<String, Object>) list.get(0));
+      return mapper.readValue(jsonString, SystemSetting.class);
+    } catch (IOException e) {
+      ProjectLogger.log(e.getMessage(), e);
```

Is this wise? Swallowing exceptions will give false success value...